src/Dibi/Drivers/MySqliResult.php

```diff
@@ -41,7 +41,8 @@ public function __construct(\mysqli_result $resultSet, bool $buffered)
 	 */
 	public function __destruct()
 	{
-		if ($this->autoFree && $this->getResultResource()) {
+		if ($this->autoFree) {
+			$this->getResultResource();
```

Comment:
changes behavior

Reply:
I tested all combinations and I think not. The result of `getResultResource()` is always true, so it should be called in `if` statement.

![Snímek obrazovky 2020-05-07 v 22 39 54](https://user-images.githubusercontent.com/4738758/81342540-ad581c80-90b3-11ea-99ee-ced9d134966d.png)

Reply:
It is right.